addons/libs/packets/fields.lua

```diff
@@ -1801,6 +1801,22 @@ fields.incoming[0x036] = L{
 }
 
 enums.indi = {
```

Comment:
This entire enum needs to be removed and replaced with appropriate bit fields. This is not your fault at all, just an observation. You may as well remove it from this commit, I'll try to adjust it tonight.